pkg/nsx/services/securitypolicy/compare.go

```diff
@@ -9,7 +9,7 @@ import (
 	util2 "github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-func SecurityPolicyEqual(existingSecurityPolicy *model.SecurityPolicy, securityPolicy *model.SecurityPolicy) bool {
+func PolicyEqual(existingSecurityPolicy *model.SecurityPolicy, securityPolicy *model.SecurityPolicy) bool {
```

Comment:
The same suggestion, Maybe SecurityPolicyEqual is more meaningful and self-explanation although PolicyEqual is more generic. We had better to keep as it is.

Reply:
ok